Src/Innovative.Blazor.Components/Components/Form/InnovativeForm.razor.css

```
@@ -187,12 +187,52 @@
     min-width: unset;
 }
 
+::deep .rz-chkbox-box:after {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)